README.md

```diff
@@ -0,0 +1,10 @@
+# How to install and run Peg and Brandon's repo #
+## If you are reading this, you are really interested in pulling down the repo for our node homework ##
+### Here is how to install and run it ###
+
+* Pull down the repo from GitHub 
+* Navigate to your local repo folder
+* Type _npm install_
+* At the node prompt, type _node server.js_
+* In your browser, type _localhost:3000_
+* you should see our index page, which is __awe-inspiring__
```

Comment:
generally, you'd want to use back-tics \` for code. That way it is very clear exactly what the user should enter. `npm install`, or `node server.js` for example.